doc/man/bitcoind.1

```diff
@@ -429,6 +429,10 @@ Shrink debug.log file on client startup (default: 1 when no \fB\-debug\fR)
 .PP
 Chain selection options:
 .HP
+\fB\-mainnet\fR
```

Comment:
No need to touch this file. Will be updated before release.

Reply:
Fixed